src/components/organisms/SchedulePageModal/SchedulePageModal.tsx

```diff
@@ -106,16 +106,22 @@ export const SchedulePageModal: React.FunctionComponent<PropsType> = ({
   }, [openVenues]);
 
   const [date, setDate] = useState(0);
+  console.log(openVenues);
 
   return (
     <div>
       <div className="/modal-content /modal-content-events">
-        <h3 className="italic">One time events</h3>
+        <div style={{ display: "flex" }}>
+          <div>
+            <h3 className="italic">One time events</h3>
+          </div>
+          {!openVenues && <div className="spinner-border" />}
```

Comment:
this does not discriminate between waiting for openVenues data to return from API, and no openVenues. Consider using typeof